FlightPlanProject/FlightPlanPlugin.cs

```diff
@@ -1204,39 +549,34 @@ public bool SetNewSMA(double burnUT, double newSMA, double burnOffsetFactor)
         }
         else
         {
-            status = Status.ERROR;
-            statusText = "Set New SMA: Solution Not Found!";
-            statusTime = UT + statusPersistence.Value;
-            Logger.LogDebug(statusText);
+            FPStatus.Error("Set New SMA: Solution Not Found!");
             return false;
         }
     }
     
-    public bool MatchPlanes(double burnUT, double burnOffsetFactor)
+    public bool MatchPlanes(TimeRef time_ref, double burnOffsetFactor)
```

Comment:
Here as we use the TimeRef iside the fct, I took the initiative to change the function input.
It appeared naturally when I move the current mode to ui